text/delegatable_cdd/README.md

```diff
@@ -0,0 +1,514 @@
+- Feature Name: Delegatable anonymous credentials, CDD scheme
+- Start Date: 2019-09-06
+- RFC PR: (leave this empty)
+- Ursa PR: https://github.com/hyperledger/ursa/pull/51
+- Version: (use whole numbers: 1, 2, 3, etc)
+
+# Summary
+[summary]: #summary
+
+This is an implementation of delegatable anonymous credentials as described in
+the CCS 17 paper [Practical UC-Secure Delegatable Credentials with Attributes
+and Their Application to
+Blockchain](https://acmccs.github.io/papers/p683-camenischA.pdf) by Camenisch
+et.al. This allows issuers to delegate issuance rights to other entities such
+that the presentations from their issued credentials do not reveal their
+identities but only the identity of the the first issuer in the delegation chain.
+This also allows issuers at each level to add attributes in their issued
+credentials. However, downstream issuers in the delegation do learn about all
+the upstream issuers, i.e. there is no privacy among issuers. This
+implementation is not UC-secure but only implements the delegatable credentials
+from the paper.
+
+# Motivation
+[motivation]: #motivation
+
+Delegatable credentials are useful in saving operational costs of issuance and
+can also make issuance cheaper. If an organization that wants to issue
+credentials has only the head of the organization (or the relevant individual)
+issuing the credential, it becomes a bottleneck. But if the head decides to
+delegate the issuance rights to its subordinates, then the credential receivers
+can check that credential comes from a delegatee of the head. In addition,
+delegation provides privacy to issuers who might want to remain anonymous to
+verifiers. This [Aries RFC](https://github.com/hyperledger/aries-rfcs/pull/218)
```

Comment:
Please update this link to: https://github.com/hyperledger/aries-rfcs/tree/master/concepts/0104-delegatable-credentials (the PR has now been merged).